resources/robots/fleetA/fleetA.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I know this is just an example file but we should eventually re-use this for the `mir100` fleet instead of `fleetA`

Reply:
Yeah, I'll change the name to mir100